### PR TITLE
test: use .env for testing

### PR DESCRIPTION
**Pull request by Reasno**

### CODE REVIEW

**Comment by Reasno** [comment]
migrate from build tags to env: https://peter.bourgon.org/blog/2021/04/02/dont-use-build-tags-for-integration-tests.html

File: otkafka/setup_test.go [hunk 1]
```diff
@@ -9,15 +10,21 @@ import (
...
+
 	os.Exit(m.Run())
 }
 
 func setupTopic() {
 	var topics = []string{"trace", "test", "example"}
 
 	for _, topic := range topics {
-		conn, err := kafka.Dial("tcp", "localhost:9092")
+		conn, err := kafka.Dial("tcp", os.Getenv("KAFKA_ADDR"))
```
Comment (lingwei0604) [inline otkafka/setup_test.go:27]: If it's cluster. is that ok ?

Comment (Reasno) [reply otkafka/setup_test.go:27]: The plan of this PR is to support env for testing. Maybe cluster support and full feature env controls can be added in seprate PRs

Comment (lingwei0604) [reply otkafka/setup_test.go:27]: ok. GOOD WORK.

**Review by lingwei0604 — APPROVED** [review]

**Review by GGXXLL — APPROVED** [review]

**Pull request merged**